components/light/rgb.rst

```diff
@@ -35,6 +35,32 @@ The ``rgb`` light platform creates an RGB light from 3 :ref:`float output compon
         pin: D1
       # Repeat for green and blue output
 
+Color Correction
+----------------
+
+It is often favourable to calibrate/correct the color produced by an LED strip light as the
+perceived intensity of different colors will generally vary. This can be done by using
+:ref:`max_power` on individual output channels:
```

Comment:
```suggestion
:ref:`max_power <config-output>` on individual output channels:
```

max_power is not a ref link, so it can't be linked to. Unfortunately, list items can't be tagged as link targets either, so the best we can do is link to the general output header, where max_power is located.